vsts/pipelines/baseImages/dotnetcore.yml

```diff
@@ -1,8 +1,13 @@
 variables:
-    ascName: oryx-automation-service-principal
-    acrName: oryxdevmcr.azurecr.io
-    skipComponentGovernanceDetection: true
-    Packaging.EnableSBOMSigning: true
+- group: Oryx
+- name: ascName
+  value: oryx-automation-service-principal
+- name: acrName
+  value: oryxdevmcr.azurecr.io
+- name: skipComponentGovernanceDetection
+  value: true
+- name: Packaging.EnableSBOMSigning
+  value: true
```

Comment:
nit: do we need to change the format of all of these existing variables to add the Oryx group? Would we not be able to do something like the following:

```yaml
variables:
  group: Oryx
  ascName: oryx-automation-service-principal
  acrName: oryxdevmcr.azurecr.io
  skipComponentGovernanceDetection: true
  Packaging.EnableSBOMSigning: true
```